Fix_git_init/Fix_git_init_home_directory.md

```diff
@@ -0,0 +1,31 @@
+﻿## **How to fix git init on home directory**
+
+You accidently ran a **`git init`** on the home directory (the wrong directory root) ? 
+Moreover, you realize that when you make a **`git status`** all the folders on your computer appear like my friend Paolo ?
+No worries dude :)

```

Comment:
This is not really a level of language we expect here.

Reply:
Sorry for this, I'll try to be more formal